main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsAgencySchema.java

```diff
@@ -0,0 +1,35 @@
+package org.mobilitydata.gtfsvalidator.table;
+
+import org.mobilitydata.gtfsvalidator.annotation.*;
+
+import java.util.Locale;
+import java.util.TimeZone;
+
+
+@GtfsTable("agency.txt")
+public interface GtfsAgencySchema extends GtfsEntity {
+    @FieldType(FieldTypeEnum.ID)
+    @PrimaryKey
+    String agencyId();
+
+    @Required
+    String agencyName();
+
+    @FieldType(FieldTypeEnum.URL)
+    @Required
+    String agencyUrl();
+
+    @Required
+    TimeZone agencyTimezone();
```

Comment:
Should this be `@FieldType(FieldTypeEnum.TIMEZONE)` as well?

Reply:
Annotation processor deducts field type in most cases (documented in FieldTypeEnum.java):

 * Many GTFS types are deducted from the actual Java types in schema definition:
 * <p>
 * * {@code int} - {@code INTEGER};
 * * {@code double} - {@code FLOAT};
 * * {@code String} - {@code TEXT};
 * * {@code boolean} - {@code BOOLEAN};
 * * {@code GtfsColor} - {@code COLOR};
 * * {@code GtfsDate} - {@code DATE};
 * * {@code GtfsTime} - {@code TIME};
 * * {@code TimeZone} - {@code TIMEZONE};
 * * {@code Locale} - {@code LANGUAGE_CODE};
 * * {@code Currency} - {@code CURRENCY_CODE}.